xmppserver/src/main/java/org/jivesoftware/openfire/server/OutgoingSessionPromise.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2005-2008 Jive Software, 2017-2024 Ignite Realtime Foundation. All rights reserved.
+ * Copyright (C) 2005-2008 Jive Software, 2017-2025 Ignite Realtime Foundation. All rights reserved.
```

Comment:
The changes in this file appear to be unrelated, is that the case?

Reply:
Yes, that's true. It's what I first ran into when trying to analyze the problem.

Those changes should be non-functional changes, with only some improvement to logging. If I were a very good boy, it'd split them off into a separate PR, but given the expected low-to-no impact and my level of laziness, I didn't.